cassandra/io/asyncioreactor.py

```diff
@@ -173,7 +180,7 @@ def push(self, data):
 
     async def _push_msg(self, chunks):
         # This lock ensures all chunks of a message are sequential in the Queue
-        with await self._write_queue_lock:
+        async with self._write_queue_lock:
```

Comment:
+1 `async with` seems to be the correct syntax to use that lock in terms of an asyncio context manager.